language/ir-testsuite/tests/scripts/script_too_few_type_args_inner.mvir

```diff
@@ -0,0 +1,5 @@
+//! type-args: 0x0::LibraAccount::Balance
```

Comment:
Good catch. The reason I'm not doing it right now is that this will require me to handle account aliases in the type arguments, which is a little bit tricky and I want to think about it a little bit more. I will put up a separate PR for this.

Reply:
Ah, that makes sense. Question: is `{{default}}` always the same address? That might be one other way out of this without having to adapt the type arguments. For example, if `{{default}}` is always `0x123`, you can write `0x123::M::R`.

Reply:
It is the same address. We now use a fixed seed for keypair generation. Personally I'd still prefer fixing the parser as the approach you mentioned is still fragile -- upstream changes in the random number generator could break such tests in a very confusing way.